wordpress-mobile/WordPress-iOS
# Remove unused PostPreviewGenerator file

To test:
Run project and make sure it compiles. 
run tests and make sure all pass
PR submission checklist:
